### DISCUSSION TITLE
Multiple-line selection doesn't work with new code search

### DISCUSSION THREAD

**Discussion by andrey-kuprianov** [community/community#39446] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Selecting multiple lines doesn't work for me after new code search got activated.

How it used to work before:

1. on any code base, click on the line number on the right -- the line is highlighted yellow.
2. click on another line, while holding `Shift`: the range of lines gets highlighted
3. click on the `...` to the left of the range, and the actions appear, like `Copy lines`, `Copy permalink`, etc.

What happens now:
After step 2, the lines are highlighted as before. But when clicking on `...` in step 3, the selection gets cleared, and only a single line is highlighted. Thus, I am not able to do actions which are specific to several lines of code.

Could you please look at that?
Thanks a lot!

**Comment by look** [comment]
Thanks for the report! We're working on a fix for this frustrating bug.

**Comment by archerzz** [comment]
@andrey-kuprianov A work-around is that after you select multiple lines, the browser address bar already has the URL with `#Lxx-Lyy`, and you can copy paste the URL in the address bar...
